runtimes/alibi-explain/mlserver_alibi_explain/runtime.py

```diff
@@ -119,7 +121,8 @@ def _load_from_uri(self, predictor: Any) -> Explainer:
         uri = model_parameters.uri
         if uri is None:
             raise InvalidModelURI(self.name)
-        return load_explainer(uri, predictor=predictor)
+        absolute_uri = await get_model_uri(self.settings)
```

Comment:
thanks. this has fallen through the cracks and got unnoticed.

Reply:
perhaps remove 
```uri = model_parameters.uri
        if uri is None:
            raise InvalidModelURI(self.name)``` as not needed anymore?

Reply:
but its optional so might still be None?

Reply:
I should have been precise. It is already being checked in `get_model_uri`.